humans/Human[] item
float64      center_x
float64      center_y
float64      confidence